holoviews/core/spaces.py

```diff
@@ -1221,6 +1220,31 @@ def dynamic_relabel(obj, **dynkwargs):
         return relabelled
 
 
+    def split_overlays(self):
+        """
+        Splits a DynamicMap into its components. Only well defined for
+        DynamicMap with consistent number and order of layers.
+        """
+        if not len(self):
+            raise ValueError('Cannot split DynamicMap before it has been initialized')
+        elif not issubclass(self.type, CompositeOverlay):
+            return [None], self
+
+        from ..util import Dynamic
+        keys = list(self.last.data.keys())
+        dmaps = []
+        for key in keys:
+            def cb(obj, overlay_key=key, **kwargs):
+                specs = [(i, util.get_overlay_spec(obj, k, v))
+                         for i, (k, v) in enumerate(obj.data.items())]
+                match = util.closest_match(overlay_key, specs)
+                return obj.data[match]
+            dmap = Dynamic(self, streams=self.streams, operation=cb)
```

Comment:
As ``operation`` is a parameter, it will be possible to get a handle on this callback after the dynamic maps are returned. For this reason, it is worth giving this callback a better name e.g ``dynamic_split_overlays`` or similar.